processing/src/main/java/org/apache/druid/query/vector/VectorCursorGranularizer.java

```diff
@@ -102,6 +71,37 @@ public static VectorCursorGranularizer create(
     return new VectorCursorGranularizer(cursor, bucketIterable, timeSelector);
   }
 
+  // And a cursor that has been made from it.
```

Comment:
This comment didn't make sense before, and it still doesn't make sense ;P

Reply:
heh, will fix :+1:

Reply:
removed